sjb/config/test_cases/test_branch_ansible_service_broker.yml

```diff
@@ -0,0 +1,109 @@
+      title: "build Origin"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        OS_BUILD_ENV_PULL_IMAGE=true OS_BUILD_ENV_PRESERVE=_output/local/bin/linux/amd64 hack/env make build
+    - type: "script"
+      title: "build Service Catalog binary"
+      repository: "service-catalog"
+      timeout: 1800
+      script: |-
+        hack/env make build
+    - type: "script"
+      title: "build Service Catalog image"
+      repository: "service-catalog"
+      timeout: 1800
+      script: |-
+        hack/env hack/build-source-image.sh
+        hack/env make build-images
+    - type: "script"
+      title: "build origin images"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        hack/build-local-images.py
+    - type: "script"
+      title: "start cluster with Service Catalog"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        if [[ "${PULL_BASE_REF:-master}" == "release-3.9" ]]; then
+          ./_output/local/bin/linux/amd64/oc cluster up --loglevel=5 --version=latest --service-catalog
+        else
+          ./_output/local/bin/linux/amd64/oc cluster up --loglevel=5 --tag=latest --enable=service-catalog
+        fi
+        ./_output/local/bin/linux/amd64/oc login -u system:admin
+        ./_output/local/bin/linux/amd64/oc describe po --all-namespaces
+    - type: "script"
+      title: "Install ASB to cluster"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        ./_output/local/bin/linux/amd64/oc create project ansible-service-broker
+        ./_output/local/bin/linux/amd64/oc process 'https://raw.githubusercontent.com/openshift/ansible-service-broker/master/templates/deploy-ansible-service-broker.template.yaml' -p BROKER_CA_CERT=$(oc get secret -n kube-service-catalog -o go-template=\'{{ range .items }}{{ if eq .type \"kubernetes.io/service-account-token\" }}{{ index .data \"service-ca.crt\" }}{{end}}{{\"\\n\"}}{{end}}\' | tail -n 1) | oc create -n ansible-service-broker -f -'
+    - type: "script"
+      title: "Relist Catalog to pickup new clusterserviceclasses"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        # relistRequets is intially zero, and to trigger relist, need to increment. Patch it to force 1.
+        ./_output/local/bin/linux/amd64/oc patch clusterservicebroker ansible-service-broker -p '{"spec":{"relistRequests":1}}'
+    - type: "script"
+      title: "Configure ASB_CI_ROOT and go build environment"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        export ASB_CI_ROOT=/data/.asb-ci
+        mkdir -p $ASB_CI_ROOT/{src,bin}
+        export GOPATH=$ASB_CI_ROOT
+        export GOBIN=$ASB_CI_ROOT/bin
+        export PATH="$PATH:GOBIN"
+    - type: "script"
+      title: "Install ASB CI tool"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        go get github.com/rthallisey/service-broker-ci/cmd/ci
+    - type: "script"
+      title: "Install ASB source"
```

Comment:
If this is the code under test you don't need to clone it manually. If it's not, let's use `sync:`

Reply:
It is the code under test; can you point me towards an example that shows how I should be cloning it?

Reply:
Like https://github.com/openshift/aos-cd-jobs/blob/6e5814f8a2fbdab27b95c5bd02ddd0b9dcc691ea/sjb/config/test_cases/test_pull_request_origin_extended_conformance_azure_39.yml#L4-L10

Reply:
Err as in the above link -- each explicitly listed item under `sync:` is code that is not directly under test -- the test is written for Origin PRs. In this job, therefore, if you are checking out ASB source in a job on the ASB repo, you can just write `sync: []`. If ASB source is being checked out in an Origin job, explicitly list it

Reply:
Sorry, I'm still not understanding what the ask is and I'm unfamiliar with any of the aos-ci stuff. What is the difference between adding to the `sync` list and "explicitly listing it"? The intention of this job is to nightly run the latest asb code with the new catalog source so we can hopefully be notified of breaking changes in the catalog sooner.